src/abi.md

```diff
@@ -1,21 +1,209 @@
 # Application Binary Interface (ABI)
 
-This section documents features that affect the ABI of the compiled output of
-a crate.
+r[abi]
 
-See *[extern functions]* for information on specifying the ABI for exporting
-functions. See *[external blocks]* for information on specifying the ABI for
-linking external libraries.
+## ABI Compatibility
+
+r[abi.compatibility]
+
+r[abi.compatibilty.type]
+Two types, `T` and `U`, can be *abi compatible*.
+
+r[abi.compatibility.equivalence]
+Two types `T` and `U` are *abi compatible* if:
```

Comment:
My first reaction upon seeing these three bullets was to say "yes this yields an equivalence relation, but not an interesting one". Skimming down I was reminded that there were going to be other base cases in the inductive definition, which is what makes it interesting.

So: Is there a (concise) way to emphasize that this is just an "if" and not an "if and only if" ?